Resources/Private/Templates/Content/Error.html

```diff
@@ -0,0 +1,56 @@
+{namespace flux=FluidTYPO3\Flux\ViewHelpers}
+<f:layout name="Content"/>
+<f:section name="Configuration">
+	<flux:form id="default">
```

Comment:
Just in the unlikely case that someone needs to target this one...

`id="error`